packages/checkout/components/totals/item/index.tsx

```diff
@@ -3,14 +3,14 @@
  */
 import classnames from 'classnames';
 import { isValidElement } from '@wordpress/element';
-import FormattedMonetaryAmount from '@woocommerce/base-components/formatted-monetary-amount';
 import type { ReactElement, ReactNode } from 'react';
 import type { Currency } from '@woocommerce/price-format';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
+import FormattedMonetaryAmount from '../../../../components/formatted-monetary-amount';
```

Comment:
For my learning why we're not able to import this as below?
```
import { FormattedMonetaryAmount } from '@woocommerce/blocks-components';

```

Reply:
That's because it causes a circular dependency, since this file is in the `@woocommerce/blocks-checkout` package, loading from the `@woocommerce/blocks-components` package will cause the `packages/components/index.js` file to load, which in turn loads `packages/components/checkbox-list` which in turn imports `CheckboxControl` from `packages/checkout/components` which in turn loads `packages/checkout/components/totals/item/index.tsx` 😁 

Hope that's easy to understand, if not feel free to load the `packages/checkout/components/totals/item/index.tsx` file and follow the dependency as if it were being loaded from the components package (`packages/components/index.js`) and see where it leads you.

Reply:
Gotcha! Thank you for detailed explanation. 🙂